CONTRIBUTORS.md

```diff
@@ -16,3 +16,4 @@ Also see the [GitHub contributor stats](https://github.com/functional-streams-fo
 - Rúnar Ó. Bjarnason ([@runarorama](https://github.com/runarorama)): Various processes and combinators, circular buffers, compression. I'm an ideas man, Michael.
 - Jed Wesley-Smith ([@jedws](https://github.com/jedws)): really very minor tweaks, cleanups and pestering, hardly worth the mention
 - Michael Pilquist ([@mpilquist](https://github.com/mpilquist)): 0.9 redesign work, maintenance
+- Daniel Urban ([@durban](https://github.com/durban)): queue peek implementation
```

Comment:
Add to build.sbt too so it shows up in the generated POM.

Reply:
Ok, thanks.

Reply:
lol I need to remember to do that as well at some point